### PR TITLE
go to region on form submission / clear

### PR DESCRIPTION
**Pull request by colinbowen**
## Changes in this PR
- Go to #browse-records id on form submission / clear for browse pages & search-transferring-body page.

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-539

## Screenshots of UI changes
N/A

### After

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/templates/main/browse-all.html [hunk 1]
```diff
@@ -69,10 +70,10 @@
                             </tr>
                             <tr class="govuk-table__row browse__table__all_desktop__row">
                                 <td class="govuk-table__cell browse__table__mobile--hidden browse__table__all_desktop">
-                                    <a href="{{ url_for('main.browse_transferring_body', _id=record['transferring_body_id']) }}">{{ record["transferring_body"] }}</a>
+                                    <a href="{{ url_for('main.browse_transferring_body', _id=record['transferring_body_id'], _anchor='browse-records') }}">{{ record["transferring_body"] }}</a>
```
Comment (anthonyhashemi) [inline app/templates/main/browse-all.html:73]: do we want to go to the browse-records section in these scenarios, do we not just want to do it when applying or clearing filters?

Comment (colinbowen) [reply app/templates/main/browse-all.html:73]: Checked with Terry on the UX for this as I wasn't sure myself. It should be applied for:

- Links within browse table
- Applied filters
- Cleared filters

**Review by pgandhizaizi — APPROVED** [review]
looks good

File: app/templates/main/browse-all.html (hunk 1 above)
Comment (anthonyhashemi) [reply app/templates/main/browse-all.html:73]: sweet 👍

**Pull request merged**